ports/psoc6/Makefile

```diff
@@ -202,12 +237,9 @@ $(MPY_MAIN_BUILD_DIR)/firmware.hex: $(MPY_MAIN_BUILD_DIR)/firmware.elf
 # include adapter makefile
 include mtb-libs/makefile_mtb.mk
 
-# include py core make definitions
-include $(TOP)/py/mkrules.mk
-
 MPY_CROSS_FLAGS += -march=armv7m
 
-build: mtb_get_build_flags $(MPY_PATH_TO_MAIN_MAKEFILE) $(MPY_MAIN_BUILD_DIR)/firmware.hex
+build: mtb_get_build_flags make_pins_csv $(MPY_PATH_TO_MAIN_MAKEFILE) $(MPY_MAIN_BUILD_DIR)/firmware.hex clean_csv
```

Comment:
why clean_csv after build?

Reply:
We don't need to retain the csv files as part of our sources. These are only intermediate files used by make-pins.py to generate the sources in build. But as soon as that is done, we don't need to save or maintain these csv files. I tried to clear them out right after make_pins target is invoked but that does not work because it complains to me as shown in image.
![image](https://github.com/Infineon/micropython/assets/96177627/c5a64721-fbbf-4758-b301-fde97faccd65)
And I understand that why. qstr generation is eventually dependent on pins.csv and even before pins_qstr.h is generated csv files are removed. But I don't know how to fix that order anymore :|

Reply:
We will refine a bit the makefile later. It is fine for now 👍

Reply:
Yes we need to.